rs-commerce/src/views/registration/components/RegistrationForm/components/AccautnRegistration/AccountRegistration.tsx

```diff
@@ -0,0 +1,51 @@
+import InputRegistration from '../InputRegistration/InputRegistration';
+import classes from './style.module.css';
+
+const inputFieldsArray = [
+  <InputRegistration
+    input={{
+      htmlFor: 'email',
+      title: 'Email:',
+      type: 'email',
+      placeholder: 'your email',
+    }}
+  />,
+  <InputRegistration
+    input={{
+      htmlFor: 'password',
+      title: 'Password:',
+      type: 'password',
+      placeholder: 'your password',
+    }}
+  />,
+  <InputRegistration
+    input={{
+      htmlFor: 'firstName',
+      title: 'First name:',
+      type: 'text',
+      placeholder: 'your name',
+    }}
+  />,
+  <InputRegistration
+    input={{
+      htmlFor: 'lastName',
+      title: 'Last name:',
+      type: 'text',
+      placeholder: 'your last name',
+    }}
+  />,
+  <InputRegistration
+    input={{
+      htmlFor: 'dateOfBirth',
+      title: 'Date of birth:',
+      type: 'date',
+      placeholder: '',
+    }}
```

Comment:
we don't need components array. create array like 
`[{   htmlFor: 'lastName',
      title: 'Last name:',
      type: 'text',
      placeholder: 'your last name',
    }, {htmlFor: 'dateOfBirth',
      title: 'Date of birth:',
      type: 'date',
      placeholder: '',}]`

and map it `{inputFieldsArray.map((item) => <InputRegistration
    input={{
      htmlFor: item.htmlFor,
      title: item.title,
      type: item.type,
      placeholder: item.placeholder,
    }}
  />)},`